cocos/game/game.ts

```diff
@@ -763,7 +765,7 @@ export class Game extends EventTarget {
                 }
                 screen.init();
                 garbageCollectionManager.init();
-                deviceManager.init(this.canvas, bindingMappingInfo);
+                await deviceManager.init(this.canvas, bindingMappingInfo);
```

Comment:
I think `await` might not work on native platform.
Does jsb support `Promise`? @dumganhar

Reply:
I looked at the logic of device-manager.ts:

```ts
            if (JSB && (globalThis as any).gfx) {
                this._gfxDevice = gfx.DeviceManager.create(deviceInfo);
            } else {
```

It seems that gfx.DeviceManager.create is a sync method on native, so `await deviceManager.init`  here may not break native platforms.

Reply:
Currently, we don't support to create JS Promise object in C++. `se` layer doesn't support that now.

Reply:
I see. We need to keep `deviceManager.init` a sync function.